browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -68,6 +73,19 @@ int OnBeforeURLRequest_StaticRedirectWork(
     return net::OK;
   }
 
+  if (translate_pattern.MatchesURL(ctx->request_url)) {
+    replacements.SetQueryStr(ctx->request_url.query_piece());
+    replacements.SetPathStr(ctx->request_url.path_piece());
+    ctx->new_url_spec =
+      GURL(kBraveTranslateEndpoint).ReplaceComponents(replacements).spec();
+    return net::OK;
+  }
+
+  if (translate_language_pattern.MatchesURL(ctx->request_url)) {
+    ctx->new_url_spec = GURL(kBraveTranslateLanguageEndpoint).spec();
```

Comment:
Looks like there are `query` and `path` piece associated with `TranslateLanguageEndpoint`, may be we should add that similar to the `translate_pattern` above.

Reply:
We won't need the query & path piece used by google endpoint for getting the support translate language from MS, so I just drop them here.